### DISCUSSION TITLE
GraphQL: Expose more fields than just the url.

### DISCUSSION THREAD

**Discussion by teamteak** [sebastian-lenz/craft-linkfield#216] [General]
Related to question https://github.com/sebastian-lenz/craft-linkfield/issues/158 - When do you plan to release 2.x? Do you need assistance with testing?  This would be a great feature to have in our headless cms.

Thanks!

**Comment by sebastian-lenz** [comment]
We've been quite there! But now Craft 4 is looming and I don't want to have people to have to deal both with a big Craft and plugin migration. So I guess we'll have to wait some more weeks.

My biggest worries about version 2 is actually the migration process, not the plugin itself. As long as I see issues like #213 popping up right here, I'll prefer to wait a bit. So if you want to help, ensuring that the migration works ist most important part.

As this is not an issue, I#ll move this topic to the discussions.
